mirage/config.js

```diff
@@ -14,32 +13,24 @@ export default function () {
   this.namespace = 'eholdings';
 
   // Package resources
-  this.get('/packages', ({ packages }, request) => {
-    return packages.all().filter((packageModel) => {
-      const query = request.queryParams.search.toLowerCase();
-      return packageModel.packageName.toLowerCase().includes(query);
-    });
-  });
+  this.get('/packages', ({ packages }, request) => packages.all().filter((packageModel) => {
+    const query = request.queryParams.search.toLowerCase();
+    return packageModel.packageName.toLowerCase().includes(query);
+  }));
 
-  this.get('/vendors/:vendorId/packages/:packageId', ({ packages }, request) => {
-    return packages.findBy({
-      vendorId: request.params.vendorId,
-      id: request.params.packageId
-    });
-  });
+  this.get('/vendors/:vendorId/packages/:packageId', ({ packages }, request) => packages.findBy({
+    vendorId: request.params.vendorId,
+    id: request.params.packageId
+  }));
 
-  this.get('/vendors/:vendorId/packages/:packageId/titles', ({ customerResources }, request) => {
-    return customerResources.where({ packageId: request.params.packageId });
-  });
+  this.get('/vendors/:vendorId/packages/:packageId/titles', ({ customerResources }, request) => customerResources.where({ packageId: request.params.packageId }));
```

Comment:
Can we just disable this eslint rule? having _really_ long lines like this is way worse imo.

Reply:
I'm fine with turning off `arrow-body-style` - now both these are back to more legible code over multiple lines.